mod/notifications.php

```diff
@@ -151,8 +148,8 @@ function notifications_content(&$a) {
 
 		// The link to switch between ignored and normal connection requests
 		$notif_show_lnk = array(
-			'href' => ($all === 0 ? 'notifications/intros/all' : 'notifications/intros' ),
-			'text' => ($all === 0 ? t('Show Ignored Requests') : t('Hide Ignored Requests'))
+			'href' => ($all === false ? 'notifications/intros/all' : 'notifications/intros' ),
+			'text' => ($all === false ? t('Show Ignored Requests') : t('Hide Ignored Requests'))
```

Comment:
Like above. When comparing boolean parameters one should always make a "if ($value)" or "if (!$value)" but never a "if ($value == true)" or so.